arviz/tests/test_plots.py

```diff
@@ -6,7 +6,7 @@
 import numpy as np
 import pytest
 import pymc3 as pm
-
+import arviz as az

```

Comment:
This should be a relative import from arviz, not an absolute import

Reply:
pylint keeps throwing the following exception: Inference Data has no 'posterior' member. Any explanation as to why this happens?

Reply:
Its because `az.InferenceData` has no posterior member :) at least during static code introspection.

If you look at the code for InferenceData posterior isn't explicitly called out, it's generated during initialization of the class. But pylint only statically analyzes code so when it "looks" it can't find anything and throws the exception.

In our case that error should be surpressed. The reason you in particular are getting so many issues is because pylint released a new version in the last couple of days, and now this PR is getting hit with all the linting errors. 

My suggestion to you is to ignore the linting errors in CI for now. Just make sure it passes in your local environment. Someone will likely make a PR in the next couple of days that handles all the new linting rules, at which point it'll get merged, and this PR can merge or rebase in the new changes. Hope that helps

https://github.com/arviz-devs/arviz/blob/master/arviz/data/inference_data.py#L24

Reply:
The tests do pass on my local environment. Thank you for the explanation. In the meantime, I'll start working on a different issue.